x-pack/plugins/cases/server/saved_object_types/migrations/utils.ts

```diff
@@ -1,73 +0,0 @@
-/*
```

Comment:
These were moved to `x-pack/plugins/cases/server/services/user_actions/transform.ts`